SDKBuildScripts/unity_RunAutoTests.sh

```diff
@@ -132,6 +137,82 @@ ExecXboxOnConsole() {
     . "$WORKSPACE/JenkinsSdkSetupScripts/JenkinsScripts/Consoles/xbox/unity_xbox.sh"
 }
 
+TryBuildAndTestAndroid() {
+    echo "TestAndroid: $TestAndroid"
+    if [ "$TestAndroid" = "true" ]; then
+        echo === Build and Test Android ===
+        pushd "${ProjRootPath}/${SdkName}_TC"
+
+            #copy test title data in
+            pushd "$WORKSPACE/SDKGenerator/SDKBuildScripts"
+                ./unity_copyTestTitleData.sh "$WORKSPACE/sdks/UnitySDK/Testing/Resources" copy
+            popd
+            if [[ $? -ne 0 ]]; then return 1; fi
+            
+            #build the APK
+            $UNITY_VERSION -projectPath "$WORKSPACE/$UNITY_VERSION/${SdkName}_TC" -quit -batchmode -executeMethod PlayFab.Internal.PlayFabPackager.MakeAndroidBuild -logFile "$WORKSPACE/${SdkName}/buildPackageOutput.txt" || (cat "$WORKSPACE/${SdkName}/buildAndroidOutput.txt" && return 1)
```

Comment:
Has _TC always been the android build folder? Or should unity_SetupTestProjects.sh build a special directory just for AppCenter?

Reply:
It has, but I never really understood what those were for.  I shall ask @pgilmorepf!

Oh wait. I guess I just did.  @pgilmorepf ?